sw/device/silicon_creator/lib/drivers/meson.build

```diff
@@ -287,6 +287,27 @@ sw_silicon_creator_lib_driver_rnd = declare_dependency(
   ),
 )
 
+test('sw_silicon_creator_lib_driver_rnd_unittest', executable(
```

Comment:
Don't bother adding meson tests. It'll slow down CI and Bazel is going to run it anyways.